Tests/GRPCNIOTransportHTTP2Tests/HTTP2TransportTLSEnabledTests.swift

```diff
@@ -14,23 +14,22 @@
  * limitations under the License.
  */
 
-import Crypto
 import Foundation
 import GRPCCore
 import GRPCNIOTransportHTTP2Posix
+import GRPCNIOTransportHTTP2TransportServices
 import NIOSSL
-import SwiftASN1
+import Network
 import Testing
-import X509
 
 @Suite("HTTP/2 transport E2E tests with TLS enabled")
 struct HTTP2TransportTLSEnabledTests {
   // - MARK: Tests
 
   @Test(
     "When using defaults, server does not perform client verification",
-    arguments: [TransportKind.posix],
-    [TransportKind.posix]
+    arguments: [TransportKind.posix, .transportServices],
+    [TransportKind.posix, .transportServices]
```

Comment:
I think you'll need a `.supported` `static` on `TransportKind` which returns `[.posix, .transportServices]` on Darwin and `[.posix]` elsewhere